### PR TITLE
#392 page notification in webapp

### PR DESCRIPTION
**Pull request by leroy-p**
#392

### CODE REVIEW

**Review by michau-k — APPROVED** [review]

File: webapp/app/components/Notifications/Form/Users/AllUsers/SelectAll/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  selectAll(event) {
+    this.props.removeUsers(this.props.users, this.props.users);
+    this.props.addUsers(this.props.users);
+  }
+
+  render() {
+    return (
+      <ButtonToolbar className={styles.toolbar}>
+        <LinkContainerButton buttonBsStyle="default" buttonText="Select all" onClick={(event) => this.selectAll(event)} />
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/Users/AllUsers/SelectAll/index.js]: ButtonBsStyle should of 'info' type and not 'default' -> default type is reserve to cancel things.

Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/Users/AllUsers/SelectAll/index.js]: ButtonBsStyle should of 'info' type and not 'default' -> default type is reserve to cancel things.

File: webapp/app/components/Notifications/Form/Users/SelectedUsers/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+//
+// Component selected users form nitifications
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/Users/SelectedUsers/index.js]: 'nitifications' should be 'notifications'

File: webapp/app/components/Notifications/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//
+// Component create group form
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/index.js]: Not the right comment

File: webapp/app/containers/Notifications/Form/Buttons/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+
+//
+// Container create group form buttons
```
Comment (clairealmozinos) [inline webapp/app/containers/Notifications/Form/Buttons/index.js]: Not the right comment

File: webapp/app/containers/Notifications/Form/Users/Buttons/index.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+
+//
+// Container users form create group
```
Comment (clairealmozinos) [inline webapp/app/containers/Notifications/Form/Users/Buttons/index.js]: Not the right comment

File: webapp/app/containers/Notifications/Form/Users/SelectedUsers/actions.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
+//
+// Selected users form create group actions
```
Comment (clairealmozinos) [inline webapp/app/containers/Notifications/Form/Users/SelectedUsers/actions.js]: Not the right comment

File: webapp/app/components/Notifications/Form/Buttons/index.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+  handleCancelClick() {
+    browserHistory.goBack();
+  }
+
+  render() {
+    return (
+      <ButtonToolbar className={styles.toolbar}>
+        <LinkContainerButton buttonType="submit" buttonBsStyle="info" buttonText="Display" onClick={(event) => this.handleCreateClick(event)} />
+        <LinkContainerButton buttonBsStyle="default" buttonText="Cancel" onClick={(event) => this.handleCancelClick(event)} />
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/Buttons/index.js]: Useless button

File: webapp/app/components/Notifications/index.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+//
+// Component Notifications
+//
+
+import React from 'react';
+import { PageHeader } from 'react-bootstrap';
+import NotificationsForm from 'containers/Notifications/Form';
+
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/index.js:8]: Missing : /* eslint-disable react/prefer-stateless-function */

File: webapp/app/containers/Notifications/Form/Title/reducer.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
+//
+// title form notifications reducer
```
Comment (clairealmozinos) [inline webapp/app/containers/Notifications/Form/Title/reducer.js]: Add a uppercase to 'title'.

**Comment by clairealmozinos** [comment]
1) "Persistance" is in French, in English it's "Persistence".
You have to change the code & the display.

2) Add '?' popover to explain what is persistance.

3) Error should be all displayed at once when use click on 'display' button

File: webapp/app/components/Notifications/Form/Buttons/index.js [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+      this.props.descriptionErrorMsg('A notification must have a description');
+      this.props.titleErrorMsg('A notification must have a title');
+    }
+  }
+
+  handleCancelClick() {
+    browserHistory.goBack();
+  }
+
+  render() {
```
Comment (clairealmozinos) [inline webapp/app/components/Notifications/Form/Buttons/index.js:29]: Change button name by 'send'. Display is not explicite

**Pull request merged**